qemu/tests/cfg/fio_perf.cfg

```diff
@@ -111,9 +111,11 @@
             order_list = "Block_size Iodepth Threads BW(MB/S) IOPS Latency(ms) Host_CPU BW/CPU KVM_Exits"
             fs_binary_extra_options = " --thread-pool-size=32"
             !Host_RHEL.m8:
-                # The below option isn't supported on RHEL8.
                 fs_binary_extra_options += " --allow-direct-io"
                 vfsd_ver_chk_cmd = "rpm -q virtiofsd"
+            Host_RHEL.m8:
+                no Host_RHEL.m8.u2 Host_RHEL.m8.u3 Host_RHEL.m8.u4 Host_RHEL.m8.u5 Host_RHEL.m8.u6 Host_RHEL.m8.u7
```

Comment:
Host_RHEL.m8.u0 and Host_RHEL.m8.u1 also support `allow_direct_io` ?

Reply:
Pls check the L89.

Reply:
Okay, got it.